games-fps/gzdoom/files/gzdoom-4.1.3-install_soundfonts.patch

```diff
@@ -0,0 +1,34 @@
+From 2d1c7ba17cac3ccd201e77ad01a9dd06ab22cb2e Mon Sep 17 00:00:00 2001
+From: William Breathitt Gray <vilhelm.gray@gmail.com>
+Date: Thu, 13 Jun 2019 18:01:08 +0900
+Subject: [PATCH] Install soundfonts and WOPL/WOPN banks
+
+The INSTALL_SOUNDFONT_PATH cache entry is used to configure the
+installation directory.
+---
+ src/CMakeLists.txt | 9 +++++++++
+ 1 file changed, 9 insertions(+)
+
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 0bb16f39e..4ff15062d 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
```

Comment:
While this is nice for upstreaming, wouldn't `doins` be simpler for Gentoo purposes?

Reply:
That is a fair point, but with this patch I can match upstream in preparation for the next release (the patch has already been merged upstream). That should allow the upcoming version bump to be a simple cp rename and removal of this patch from the `PATCHES` list; with the `doins` approach, I'll have to remove the `doins` and reimplement the `INSTALL_SOUNDFONT_PATH` definition line again for the version bump.